source/auth/auth.md

```diff
@@ -1217,10 +1217,15 @@ in the MONGODB-OIDC specification, including sections or blocks that specificall
 
   - PROVIDER_NAME\
     Drivers MUST allow the user to specify the name of a built-in OIDC provider integration to use to
-    obtain credentials. If provided, the value MUST be one of `["aws"]`. If both `PROVIDER_NAME` and an
+    obtain credentials. If provided, the value MUST be one of `["aws", "azure"]`. If both `PROVIDER_NAME` and an
     [OIDC Callback](#oidc-callback) or [OIDC Human Callback](#oidc-human-callback) are provided for the same
     `MongoClient`, the driver MUST raise an error.
 
+  - TOKEN_AUDIENCE\
```

Comment:
`OIDC_TOKEN_AUDIENCE`?

We sometimes prefix with OIDC (the callbacks), but sometimes not (provider name). This seems like a mild inconsistency - is it intentional?

Reply:
Actually, this brings up a wider point.  These auth mechanism properties are scoped under their respective auth mechanisms, and are already therefore namespaced.  To date, only `AWS_SESSION_TOKEN` is prefixed, but we are discussing an `OIDC_ENV` in #1544.  I think leaving `AWS_SESSION_TOKEN` makes sense for legacy reasons and because that is actually the name of an environment variable that is used, but I'm think that not only should this remain "TOKEN_AUDIENCE", but we should not use a prefix for the environment, but call it `ENVIRONMENT`.

Reply:
I agree that having to add a prefix to every auth mechanism property seems impractical. Based on previous conversations with @jyemin I understand some drivers may have to refactor the connection string parser to support the same auth mechanism property in multiple auth mechanisms.

Reply:
The consensus seems to be `TOKEN_RESOURCE` based on further discussion.

Reply:
That sounds good to me